src/main/java/messenger/MessengerDatabase.java

```diff
@@ -0,0 +1,43 @@
+package messenger;
+
+import notification.Notification;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.util.HashMap;
+import java.util.HashSet;
+
+public class MessengerDatabase {
+
+    private HashMap<String, String> idToNameMap;
+
+    private HashSet<Notification> storedNotifications;
+
+    public MessengerDatabase() {
+        storedNotifications = new HashSet<>();
+        try {
+            FileInputStream fis = new FileInputStream("src/main/resources/idToName.ser");
+            ObjectInputStream ois = new ObjectInputStream(fis);
+            idToNameMap = (HashMap) ois.readObject();
+            ois.close();
+            fis.close();
+        } catch (IOException ioe) {
+            ioe.printStackTrace();
+            return;
```

Comment:
don't use `return` in constructor

Reply:
Why not?

Reply:
Well, this is the end of this method anyways xD And as I said earlier, leave object creation for JVM to handle. This is memory management and stuff. Constructors should not have logic in them, no forking with ifs, no execution of code that can throw exceptions. It's only to create object